liblepton/scheme/color-map.scm

```diff
@@ -0,0 +1,50 @@
+; Lepton EDA
+; Backward-compatible color-map.scm file
+; Copyright (C) 2020 Lepton EDA Contributors
+; License: GPLv2+. See the COPYING file
+
+
+; Association list: %color-name-map
+;
+; Defines mapping between color symbols
+; like 'background, 'net, 'text, etc.
+; and integer values.
+;
+( define %color-name-map
+  ( @ (lepton color-map) %color-name-map )
+)
+
+
+; Function: color-map-name-to-index()
+;
+; Returns an integer index given one of a color
+; symbols like 'background, 'net, 'text, etc.
+; Returns its argument if wrong symbol is specified.
+;
+( define color-map-name-to-index
+  ( @ (lepton color-map) color-map-name-to-index )
+)
+
+
+; Function: color-map-name-from-index()
+;
+; Returns one of a color symbols like 'background,
+; 'net, 'text, etc., given an integer index.
+; Returns its argument if wrong index is specified.
+;
+( define color-map-name-from-index
+  ( @ (lepton color-map) color-map-name-from-index )
+)
+
```

Comment:
You can just add:
> (use-modules ((lepton color-map)
>              #:select (%color-name-map
>                        color-map-name-to-index
>                        color-map-name-from-index)))

instead of all this.
Comments could go to `(lepton color-map)` itself as comments or docstrings (it's better since users could reference them then in interactive mode).  And here could be just reference to the module, something like: Please see the code of the module bla-bla ... use the command `,describe function-name` in interactive mode to get the function description.